vendor/bat-native-ledger/BUILD.gn

```diff
@@ -117,14 +117,20 @@ source_set("ledger") {
     "//third_party/boringssl",
     "//third_party/re2",
     "//url",
-    rebase_path("bat-native-ads", dep_base),
     rebase_path("bat-native-confirmations", dep_base),
```

Comment:
Do we need to remove `bat-native-confirmations` here?

Reply:
Fixed